test/dlc_tests/ecs/tensorflow/inference/test_ecs_tensorflow_inference.py

```diff
@@ -51,23 +51,23 @@ def test_ecs_tensorflow_inference_eia(tensorflow_inference_eia, ecs_container_in
 
 
 
-@pytest.mark.model("half_plus_two")
-@pytest.mark.parametrize("ecs_instance_type", ["p2.8xlarge"], indirect=True)
+@pytest.mark.model("albert")
+@pytest.mark.parametrize("ecs_instance_type", ["p2.xlarge"], indirect=True)
 @pytest.mark.parametrize("ecs_ami", [ECS_AML2_GPU_USWEST2], indirect=True)
 def test_ecs_tensorflow_inference_gpu(tensorflow_inference, ecs_container_instance, region, gpu_only):
     worker_instance_id, ecs_cluster_arn = ecs_container_instance
     public_ip_address = ec2_utils.get_public_ip(worker_instance_id, region=region)
     num_gpus = ec2_utils.get_instance_num_gpus(worker_instance_id)
 
-    model_name = "saved_model_half_plus_two"
+    model_name = "albert"
     service_name = task_family = revision = None
     try:
         service_name, task_family, revision = ecs_utils.setup_ecs_inference_service(
             tensorflow_inference, "tensorflow", ecs_cluster_arn, model_name, worker_instance_id, num_gpus=num_gpus,
             region=region
         )
         model_name = get_tensorflow_model_name("gpu", model_name)
-        inference_result = request_tensorflow_inference(model_name, ip_address=public_ip_address)
+        inference_result = request_tensorflow_inference_nlp(model_name, ip_address=public_ip_address)
```

Comment:
Could you retain the existing test and add the new model? Feel free to introduce another test if necessary

Reply:
Retained the existing tests